robot/rospackages/src/mcu_control/scripts/PdsNode.py

```diff
@@ -205,9 +206,12 @@ def publish_pds_data(message):
             elif i < 7:
                 current.effort.append(float(dataPDS[i]))
             elif i < 10:
-                temp.x = float(dataPDS[7])
-                temp.y = float(dataPDS[8])
-                temp.z = float(dataPDS[9])
+                if i == 7:
+                    temp.x = checkTempErrorCase(float(dataPDS[i]))
+                if i == 8:
+                    temp.y = checkTempErrorCase(float(dataPDS[i]))
+                if i == 9:
+                    temp.z = checkTempErrorCase(float(dataPDS[i]))
```

Comment:
why not do this ?
```suggestion
                temp.x = checkTempErrorCase(float(dataPDS[7]))
                temp.y = checkTempErrorCase(float(dataPDS[8]))
                temp.z = checkTempErrorCase(float(dataPDS[9]))
```

Reply:
I guess its because previously it was ran 3 times? (Man that for loop is garbage with that magic `i`)

Reply:
I honestly don't know why I checked each case individually. 
Also lol yes that loop is great. Prob the person who wrote the PDS code also wrote this?

Reply:
![image](https://user-images.githubusercontent.com/18620025/82130843-f9cee680-979c-11ea-8ce8-a07fa2eccdf0.png)


Use `git blame` @AFknUsername

Reply:
@vashmata 👀 👀